### PR TITLE
fix: Fix the sh and lable is not correct in pipeline editor

### PR DESCRIPTION
**Pull request by harrisonliu5**
Signed-off-by: harrisonliu5 <harrisonliu_5@163.com>

**What type of PR is this?**

 /kind bug

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #[kubesphere/kubesphere/3950](#https://github.com/kubesphere/kubesphere/issues/3950) #[kubesphere/kubesphere/3949](#https://github.com/kubesphere/kubesphere/issues/3949)

**Special notes for reviewers**:
```
```

**Additional documentation, usage docs, etc.**:

```docs

```

/area devops

### CODE REVIEW

**Comment by harrisonliu5** [comment]
https://user-images.githubusercontent.com/17872673/121657158-be533880-cad2-11eb-9e33-b93c7e65fcf6.mp4

File: src/pages/devops/components/Pipeline/StepModals/shell.jsx [hunk 1]
```diff
@@ -46,13 +46,14 @@ export default class Shell extends React.Component {
...
       const value = get(nextProps.edittingData.data, '[0].value.value', '')
+
       return { value }
     }
     return null
   }
 
   handleChange = value => {
-    this.newValue = value
+    this.newValue = value || `""`
```
Comment (JohnNiang) [inline src/pages/devops/components/Pipeline/StepModals/shell.jsx:56]: If double quote `""` is set by default, then the Jenkinsfile output will be:

```groovy
pipeline {
  agent any
  stages {
    stage('stage-i8bjw') {
      agent {
        node {
          label 'base'
        }

      }
      steps {
        sh '""'
      }
    }

  }
}
```

After running this pipeline, the bash will complain an error: `Permission Denied`.

So I'd prefer to leave it empty string.

**Comment by LinuxSuRen** [comment]
fixes https://github.com/kubesphere/kubesphere/issues/3950
fixes https://github.com/kubesphere/kubesphere/issues/3949

File: src/pages/devops/components/Pipeline/StepModals/shell.jsx (hunk 1 above)
Comment (harrisonliu5) [reply src/pages/devops/components/Pipeline/StepModals/shell.jsx]: update

**Comment by harrisonliu5** [comment]
the code is updated @JohnNiang

**Comment by JohnNiang** [comment]
> the code is updated @JohnNiang

Thanks for your code change.

/lgtm

**Pull request merged**